app/src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,17 @@
       android:label="@string/app_name"
       android:launchMode="singleTop"
       android:windowSoftInputMode="adjustPan">
+      <intent-filter>
```

Comment:
@Frans-Lukas I want to see this happen so we clean up the accessibility of this feature

Reply:
So same as is but removing the intent-filter?

Reply:
yeah, instead of accessing this by launching a share intent we have a dedicated icon that navigates with a bundle. We may also want to keep the share icon? I don't think I have ever tried sharing a file by other means

Reply:
@Frans-Lukas this intent filter should be removed and replaced